decam/makeLinearizer.py

```diff
@@ -61,6 +62,33 @@ def makeLinearizerDecam(fromFile, force=False, verbose=False):
                 print("LSST  table for %s=%s..." % (ampName, lsstTable[i, 0:5],))
         linearizer = lsstTable
         butler.put(linearizer, "linearizer", dataId=dict(ccdnum=ccdnum))
+
+        # Generate gen3 files into local directory.
+        myLinearity = Linearizer(table=lsstTable)
+        for i, ampName in enumerate("AB"):
+            myLinearity.ampNames.append(ampName)
+            myLinearity.linearityType[ampName] = 'LookupTable'
+            myLinearity.linearityCoeffs[ampName] = np.array([i, 0])
+            myLinearity.linearityBBox[ampName] = detector.getAmplifiers()[i].getBBox()
+            myLinearity.fitParams[ampName] = np.array([])
+            myLinearity.fitParamsErr[ampName] = np.array([])
+            myLinearity.fitChiSq[ampName] = np.nan
```

Comment:
Can you please help me understand what all these initialized empty/nan things are used for?

Reply:
For linearizers with a functional form (the Squared and Polynomial methods), these are populated with the full fit parameters, which are then converted into the crosstalk coefficients by scaling by the linear term (which should be equal to 1.0).  The docstring describing this relation has moved around many times, but now lives here: https://github.com/lsst/cp_pipe/blob/master/python/lsst/cp/pipe/linearity.py#L162

Reply:
OK. Can you please update the docstring for `makeLinearizerDecam` while you're in here?